src/pages/Admin.vue

```diff
@@ -0,0 +1,77 @@
+<script lang="ts" setup>
+import { storeToRefs } from 'pinia'
+import { onMounted, ref } from 'vue'
+import 'vue-select/dist/vue-select.css'
+
+import Button from '../components/shared/Button.vue'
+import VueSelect from '../components/shared/VueSelect.vue'
+import { useAdminStore } from '../stores/admin'
+import { useUserStore } from '../stores/user'
```

Comment:
`/@`が`/src`のaliasとして登録されているので、依存をわかりやすくするため、同階層以下のものを使う時以外は`/@`を使ってください（つまり、`/src`や'../'を使わない）。
```suggestion
import Button from '/@/components/shared/Button.vue'
import VueSelect from '/@/components/shared/VueSelect.vue'
import { useAdminStore } from '/@/stores/admin'
import { useUserStore } from '/@/stores/user'
```

Reply:
下2つだけ「モジュール '@/stores/admin' またはそれに対応する型宣言が見つかりません。」になってしまいました...

Reply:
修正したものをpushしておきました。